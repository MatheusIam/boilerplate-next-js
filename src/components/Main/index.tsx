import * as S from './styles'

const Main = () => (
  <S.Wrapper>
    <h1>OlÃ¡ mundo, Matheus Iam!!</h1>
  </S.Wrapper>
)

export default Main
