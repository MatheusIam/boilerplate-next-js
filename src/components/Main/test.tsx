import React from 'react'
import { render } from '@testing-library/react'
import Main from '.'

describe('Main component', () => {
  it('renders the expected title', () => {
    const { getByText } = render(<Main />)
    const title = getByText('React Avançado')
    expect(title).toBeInTheDocument()
    expect(title.textContent).toBe('React Avançado')
  })

  it('renders the expected logo', () => {
    const { getByAltText } = render(<Main />)
    const logo = getByAltText(
      'Imagem de um átomo e React Avançado escrito ao lado.'
    )
    expect(logo).toBeInTheDocument()
  })
})
