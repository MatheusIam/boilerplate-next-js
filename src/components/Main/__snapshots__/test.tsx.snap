// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<main
  class="sc-beyTiQ kaIBlV"
>
  <h1>
    Ol√° mundo, Matheus Iam!!
  </h1>
</main>
`;
